packages/evm/contracts/registry/GuardiansRegistry.sol

```diff
@@ -207,6 +226,37 @@ contract GuardiansRegistry is ControlledRecoverable, IGuardiansRegistry, ERC900,
         _stake(_from, _from, _amount, _data);
     }
 
+    /**
+    * @notice Lock `@tokenAmount(self.token(), _amount)` from deactivation
+    * @param _lockManager Address of the lock manager that will control the lock
+    * @param _amount Amount of tokens to be locked from deactivation
+    */
+    function lockDeactivation(address _lockManager, uint256 _amount) external {
+        _lockDeactivation(msg.sender, _lockManager, _amount);
+    }
+
+    /**
+    * @notice Unlock `@tokenAmount(self.token(), _amount)` from deactivation for `_guardian`
+    * @param _guardian Address of the guardian unlocking the deactivation amount of
+    * @param _lockManager Address of the lock manager controlling the lock
+    * @param _amount Amount of tokens to be unlocked for deactivation
+    */
+    function unlockDeactivation(address _guardian, address _lockManager, uint256 _amount) external {
+        require(ILockManager(_lockManager).canUnlock(_guardian, _amount), ERROR_CANNOT_UNLOCK_DEACTIVATION);
```

Comment:
We may want to check if the `msg.sender` is the lock manager and skip this check if so.